### ISSUE TITLE
SuggesterControllerTest errors on Windows

### ISSUE THREAD

**Issue by vladak** [oracle/opengrok#2559]
after the changes in PR #2555:
```
[00:28:37] [ERROR]   SuggesterControllerProjectsDisabledTest.before:96 � ConditionTimeout Condition...
[00:28:37] [ERROR]   SuggesterControllerTest.before:130 � ConditionTimeout Condition with org.openg...
[00:28:37] [ERROR]   SuggesterControllerTest.before:130 � ConditionTimeout Condition with org.openg...
[00:28:37] [ERROR]   SuggesterControllerTest.before:130 � ConditionTimeout Condition with org.openg...

...
```

There are 34 these `ConditionTimeout` errors (not failures) on Windows.

**Comment by vladak** [comment]
There is not much information in the log:
```
[00:28:22] [ERROR] testWildcardQueryEndingWithAsterisk(org.opengrok.web.api.v1.controller.SuggesterControllerTest)  Time elapsed: 15.205 s  <<< ERROR!
[00:28:22] org.awaitility.core.ConditionTimeoutException: Condition with org.opengrok.web.api.v1.controller.SuggesterControllerTest was not fulfilled within 15 seconds.
[00:28:22]      at org.opengrok.web.api.v1.controller.SuggesterControllerTest.before(SuggesterControllerTest.java:130)
```

**Comment by vladak** [comment]
fixed in d8ea1e539aa63ba0909eacc00b476e01e5cce573
